### DISCUSSION TITLE
Alevin with 10x 5' v2

### DISCUSSION THREAD

**Discussion by JLTrincado** [COMBINE-lab/salmon#649] [Q&A (single-cell specific)] [answered]
Dear authors,

This is a question regarding 10x 5' reads. In the tutorial, it's explicitly indicated that Alevin only works with 3’ tagged-end single-cell sequencing data. On my reads, I already have filtered the Cellular Barcode(CB) + Unique Molecule identifier (UMI) in one read file and the read sequence in the other. Did you try with this kind of reads?

Congrats on this extremely useful tool.

Best,

Juan Luis.

**Comment by k3yavi** [comment, marked as answer]
Hi @JLTrincado ,

Yes, you can quantify 10x 5' data with alevin, just change the command line flag `-lISR` to `-lISF` keeping everything same and it should work out of the box.
